lib/tasks/deploy.rake

```diff
@@ -46,6 +46,12 @@ def heroku_migrate_and_restart(remote)
                 "ruby /usr/local/heroku/bin/heroku run rake loomio:notify_clients_of_update -a #{remote}"]
 end
 
+desc "Copy development client to a versioned client for production"
+task :version_client_assets do
```

Comment:
I assume one would. I don't need a deploy rake task for the docker based system though.

I though that It would be better not to touch that stuff while you're working on it.